java/vector/src/test/java/org/apache/arrow/vector/TestCopyFrom.java

```diff
@@ -1069,4 +1070,27 @@ public void testCopyFromWithNulls14() {
       }
     }
   }
+
+  @Test //https://issues.apache.org/jira/browse/ARROW-7837
+  public void testCopySafeArrow7837() {
```

Comment:
nit: a more descriptive name can be better? (Maybe adding some descriptive comments )

Reply:
added a comment